client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraph.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="provenance-graph-container" v-if="isOpen">
+    <aside slot="left">
+      <div class="btn-group btn-group-sm">
+        <button
+          class="btn btn-secondary"
+          @click="onClickCondensed"
+          > Condensed
+        </button>
+        <button
+          class="btn btn-secondary"
+          @click="onClickExpanded"
+          > Expanded
+        </button>
+      </div>
+    </aside>
+    <close-button @close="onClose"/>
```

Comment:
I don't thik this button is necessary as you can use the same button to Open and Close the provenance graph panel

Reply:
We can keep it for now since it gives the user two ways to close the panel.